.travis.yml

```diff
@@ -1,22 +1,41 @@
+sudo: false
 language: python
 python:
   - "2.7"
 # command to install wxPython using apt
-before_install:
-  - sudo apt-get update -qq
-  - sudo apt-get install -qq python-wxgtk2.8
+# before_install:
+#   - sudo apt-get update -qq
+#   - sudo apt-get install -qq python-wxgtk2.8
 # command to install other dependencies
+
+addons:
+  apt:
+    packages:
+      # for wxPython:
+      - python-wxgtk2.8
+      - python-wxtools
+      - wx2.8-doc
+      - wx2.8-examples
+      - wx2.8-headers
+      - wx2.8-i18n
```

Comment:
Do we need to install wx for testing purposes now that the retriever only loads it for the GUI following c73c39c? I suspect we can remove the wx imports entirely.